lib/client/api.go

```diff
@@ -430,6 +430,21 @@ type Config struct {
 
 	// PROXYSigner is used to sign PROXY headers for securely propagating client IP address
 	PROXYSigner multiplexer.PROXYHeaderSigner
+
+	// dtAttemptLoginIgnorePing and dtAutoEnrollIgnorePing allow Device Trust
+	// tests to ignore Ping responses.
+	// Useful to force flows that only typically happen on Teleport Enterprise.
+	dtAttemptLoginIgnorePing, dtAutoEnrollIgnorePing bool
+
+	// DTAuthnCeremony allows tests to override the default device
+	// authentication function.
+	// Defaults to [dtauthn.NewCeremony].
+	DTAuthnCeremony dtauthn.CeremonyI
+
+	// dtAutoEnroll allows tests to override the default device auto-enroll
+	// function.
+	// Defaults to [dtenroll.AutoEnroll].
+	dtAutoEnroll dtAutoEnrollFunc
```

Comment:
nits: publics before privates?

```suggestion
	// DTAuthnCeremony allows tests to override the default device
	// authentication function.
	// Defaults to [dtauthn.NewCeremony].
	DTAuthnCeremony dtauthn.CeremonyI

	// dtAttemptLoginIgnorePing and dtAutoEnrollIgnorePing allow Device Trust
	// tests to ignore Ping responses.
	// Useful to force flows that only typically happen on Teleport Enterprise.
	dtAttemptLoginIgnorePing, dtAutoEnrollIgnorePing bool

	// dtAutoEnroll allows tests to override the default device auto-enroll
	// function.
	// Defaults to [dtenroll.AutoEnroll].
	dtAutoEnroll dtAutoEnrollFunc
```